packages/dashmate/src/docker/DockerCompose.js

```diff
@@ -377,22 +374,22 @@ class DockerCompose {
       throw new Error(`Update Docker to version ${DockerCompose.DOCKER_MIN_VERSION} or higher`);
     }
 
+    let version
+
     // Check docker compose
-    if (process.env.DOCKER_COMPOSE_V2) {
-      try {
-        await execAsync('docker compose');
-      } catch (e) {
-        throw new Error('Docker Compose V2 is not installed');
-      }
-    } else {
-      if (!hasbin.sync('docker-compose')) {
-        throw new Error('Docker Compose is not installed');
-      }
+    try {
+      ({out: version} = await dockerCompose.version());
 
-      const { out: version } = await dockerCompose.version();
       if (semver.lt(version.trim(), DockerCompose.DOCKER_COMPOSE_MIN_VERSION)) {
```

Comment:
I would recommend to move out this check form try catch block otherwise you do local catching which not good